rpm/SPECS/openresty.spec

```diff
@@ -218,6 +228,10 @@ fi
 /usr/bin/resty
 %{orprefix}/bin/resty
 
+%files devel
+%defattr(-,root,root,-)
+
+/usr/include/nginx/*.h
```

Comment:
I don't think this is correct. NGINX never installs any C header files, not to mention the installation path `/usr/include/` assumed here. We should never pollute the system header directories with nginx's header files since they would never be useful to applications other than nginx/openresty.